scripts/run_example

```diff
@@ -97,28 +98,40 @@ fi
 
 readonly CLIENT_ARGS=("${@-}")  # Choose client args provided after '--'.
 
-# Run the application client.
-case "${client_language}" in
-  rust)
-    cargo run --release --target=x86_64-unknown-linux-musl --manifest-path="${SCRIPTS_DIR}/../examples/${EXAMPLE}/client/rust/Cargo.toml" -- \
-      --root-tls-certificate="${SCRIPTS_DIR}/../examples/certs/local/ca.pem" \
-      "${CLIENT_ARGS[@]-}"
-    ;;
-  cpp)
-    "./bazel-client-bin/examples/${EXAMPLE}/client/client" "${TLS_ARGS[@]}" "${ADDITIONAL_ARGS[@]-}" "${CLIENT_ARGS[@]-}"
-    ;;
-  nodejs)
-    nodejs_client="./examples/${EXAMPLE}/client/nodejs"
-    if [[ -d "${nodejs_client}" ]]; then
-        npm start --prefix "./examples/${EXAMPLE}/client/nodejs"
-    else
-      echo "The ${EXAMPLE} example does not contain a ${client_language} client."
-      exit 1
-    fi
-    ;;
-  *)
-    echo "Invalid example client variant: ${client_language}"
-    exit 1;;
-esac
+# Run the application clients.
+for client_language in $client_languages
+do
+  case "${client_language}" in
+    rust)
+      rust_client_manifest="${SCRIPTS_DIR}/../examples/${EXAMPLE}/client/rust/Cargo.toml"
+      if [[ -d "${rust_client_manifest}" ]]; then
+          cargo run --release --target=x86_64-unknown-linux-musl --manifest-path="${rust_client_manifest}" -- \
+            --root-tls-certificate="${SCRIPTS_DIR}/../examples/certs/local/ca.pem" \
+            "${CLIENT_ARGS[@]-}"
+      else
+        echo "The ${EXAMPLE} example does not contain a ${client_language} client. Skipping this client."
```

Comment:
IIUC If we will run `-c rust` on the example that doesn't support it, there will be no error, and the example script will return `0` as a status.

Reply:
Yes, that is indeed correct. It will print a warning, but it won't fail. 

If we think it's important to fail in this case, we'd need to implement a slightly different approach, such as `-c all` to run all available clients. 

What do you think?

Reply:
I don't think the checks for which client languages exist are working properly, and the fail-open nature of the checks means that our [Cloudbuild is currently running](https://pantheon.corp.google.com/cloud-build/builds/bdcc0238-2511-4076-90da-4810b7719888;step=9?project=oak-ci) almost none of the examples:
```bash
% grep "example does not contain" log-bdcc0238-2511-4076-90da-4810b7719888.txt  | grep -v ': +'
Step #9 - "run_examples": The abitest example does not contain a rust client. Skipping this client.
Step #9 - "run_examples": The abitest example does not contain a cpp client. Skipping this client.
Step #9 - "run_examples": The abitest example does not contain a nodejs client. Skipping this client.
Step #9 - "run_examples": The translator example does not contain a rust client. Skipping this client.
Step #9 - "run_examples": The translator example does not contain a cpp client. Skipping this client.
Step #9 - "run_examples": The translator example does not contain a nodejs client. Skipping this client.
Step #9 - "run_examples": The trusted_information_retrieval example does not contain a rust client. Skipping this client.
Step #9 - "run_examples": The trusted_information_retrieval example does not contain a cpp client. Skipping this client.
Step #9 - "run_examples": The trusted_information_retrieval example does not contain a nodejs client. Skipping this client.
Step #9 - "run_examples": The hello_world example does not contain a rust client. Skipping this client.
Step #9 - "run_examples": The hello_world example does not contain a cpp client. Skipping this client.
Step #9 - "run_examples": The chat example does not contain a rust client. Skipping this client.
Step #9 - "run_examples": The chat example does not contain a cpp client. Skipping this client.
Step #9 - "run_examples": The chat example does not contain a nodejs client. Skipping this client.
Step #9 - "run_examples": The running_average example does not contain a rust client. Skipping this client.
Step #9 - "run_examples": The running_average example does not contain a cpp client. Skipping this client.
Step #9 - "run_examples": The running_average example does not contain a nodejs client. Skipping this client.
Step #9 - "run_examples": The private_set_intersection example does not contain a rust client. Skipping this client.
Step #9 - "run_examples": The private_set_intersection example does not contain a cpp client. Skipping this client.
Step #9 - "run_examples": The private_set_intersection example does not contain a nodejs client. Skipping this client.
Step #9 - "run_examples": The machine_learning example does not contain a rust client. Skipping this client.
Step #9 - "run_examples": The machine_learning example does not contain a cpp client. Skipping this client.
Step #9 - "run_examples": The machine_learning example does not contain a nodejs client. Skipping this client.
Step #9 - "run_examples": The aggregator example does not contain a rust client. Skipping this client.
Step #9 - "run_examples": The aggregator example does not contain a cpp client. Skipping this client.
Step #9 - "run_examples": The aggregator example does not contain a nodejs client. Skipping this client.
Step #10 - "run_examples_cpp": The hello_world example does not contain a cpp client. Skipping this client.
Step #10 - "run_examples_cpp": The tensorflow example does not contain a cpp client. Skipping this client.
```

@juliettepretot  please could you take a look?